README.md

```diff
@@ -63,3 +63,72 @@ And poof, you are ready to make a PR with the updates.
 ## Test environments
 
 Template projects are hooked up to Platform.sh projects, so each new PR gets built as a new, empty environment ready for testing.  In most cases simply visiting the built site and verifying that the installer can run (if available) or that the site gives the appropriate "there's nothing here yet" error is sufficient, but further testing can be done if needed.
+
+
+## Contributing
+
+If you would like to contribute to the list of Platform.sh's maintained templates, there is a simple process for ensuring that future maintenance is set up from the start through `template-builder`.
+
+Let's use the previous example: you have created a new application that uses the framework Spiffy that you think would be a useful template. 
+
+1. Create a repository under the [Platform.sh Github organization](https://github.com/platformsh/) for the template. Officially maintained templates begin with the `template-` prefix, so name it `template-spiffy`, and initialize it with an empty `README.md`.
+2. Add the "Examples and Templates team" as `Admin` collaborators in the `platformsh/template-spiffy` repository settings.
+3. Set up GitHub integration for the repository. When you submit your template for review it will helpful if there is already a live site that shows what the application would look like to customers and that it deploys properly. You can find information about how to set up a [GitHub integration](https://docs.platform.sh/administration/integrations/github.html#github) in the documentation.
```

Comment:
This is missing a step to create a Psh project, and what region it should be on (EU-3).  Also, this would be a good time to discuss whose key to use for these integrations.  We should probably not always use mine. :smile:

Reply:
I guess I left that open for the moment because we hadn't decided/set up a separate account for templates yet, suggesting that new submissions should at the very least be set up on a project on THEIR account for the review process. Then we could delete the integration and substitute it for something else when we do have that set up. 

Although maybe now's the time. How long it would it take to set up a `templates@platform.sh` account?